bake/compose.go

```diff
@@ -263,3 +267,21 @@ func composeToBuildkitSecret(inp compose.ServiceSecretConfig, psecret compose.Se
 
 	return strings.Join(bkattrs, ","), nil
 }
+
+func dedupMap(ms ...map[string]string) map[string]string {
```

Comment:
We should put this next to `dedupString` - maybe in a separate file if they don't both fit in a single file neatly?

`dedupString` is a weird name as well for that now we'd have `dedupMap`, maybe we could change it to `dedupSlice` or equivalent?